sim/firesim-lib/src/main/scala/util/Configs.scala

```diff
@@ -70,3 +73,6 @@ class ILADepth16384 extends WithILADepth(16384)
 
 
 class Congestion extends WithBuildStategy(BuildStrategies.Congestion)
+class Timing extends WithBuildStategy(BuildStrategies.Timing)
+class Explore extends WithBuildStategy(BuildStrategies.Explore)
+class NoRetiming extends WithBuildStrategy(BuildStrategies.NoRetiming)
```

Comment:
```suggestion
class Congestion extends WithBuildStrategy(BuildStrategies.Congestion)
class Timing extends WithBuildStrategy(BuildStrategies.Timing)
class Explore extends WithBuildStrategy(BuildStrategies.Explore)
class NoRetiming extends WithBuildStrategy(BuildStrategies.NoRetiming)
```

Reply:
Additionally, the config. fragment needs to be changed from `WithBuildStategy` to `WithBuildStrategy`